libs/filamat/src/MaterialBuilder.cpp

```diff
@@ -54,6 +54,12 @@ namespace filamat {
 
 std::atomic<int> MaterialBuilderBase::materialBuilderClients(0);
 
+inline void assertSingleTargetApi(MaterialBuilderBase::TargetApi api) {
```

Comment:
Ash thanks for adding this assert! My personal aesthetic would've been to use strong typing instead, but this works.